app/controllers/courses_controller.rb

```diff
@@ -945,13 +950,15 @@ def write_cuds(cuds)
     rowCUDs.each do |cud|
       next unless duplicates.include?(cud[:email])
 
-      msg = "Validation failed: Duplicate email #{cud[:email]}"
-      if !rosterErrors.key?(msg)
-        rosterErrors[msg] = []
+      msg = "Warning : Duplicate email #{cud[:email]}"
```

Comment:
Can you add that this duplicate email was ignored / which line the duplicate email was on?